lib/utils/webpack.config.js

```diff
@@ -368,12 +368,12 @@ module.exports = (program, directory, suppliedStage, webpackPort = 1500, routes
         config.loader('less', {
           test: /\.less/,
           exclude: /\.module\.less$/,
-          loader: ExtractTextPlugin.extract(['css?minimize', 'less']),
+          loader: ExtractTextPlugin.extract(['css?minimize', 'postcss']),
         })
         config.loader('sass', {
           test: /\.(sass|scss)/,
```

Comment:
You removed the less/sass loaders so now sass/less code would just be run through postcss and not less/sass.

It should look like `loader: ExtractTextPlugin.extract(['css?minimize', 'postcss', 'less']),`